src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -177,7 +179,15 @@ private TestPackage CreateTestPackage(string assemblyName)
 
             // Always run one assembly at a time in process in it's own domain
             package.Settings[PackageSettings.ProcessModel] = "InProcess";
-            package.Settings[PackageSettings.DomainUsage] = "Single";
+
+            if (Settings.DomainUsage != null)
+            {
+                package.Settings[PackageSettings.DomainUsage] = Settings.DomainUsage;
```

Comment:
Our `ConsoleOptions` in the NUnit Console project constrains this to "None", "Single" and "Multiple". We should probably constrain it to valid values here to prevent our parsing in the engine from failing. Either that, or we should wrap the `Enum.Parse` in the engine in a try/catch and default to "Default" on failure.